Resources/Audio/Effects/Footsteps/attributions.yml

```diff
@@ -22,6 +22,7 @@
 
 - files:
   - slime1.ogg
+  - slime2.ogg # the sound is quieter
```

Comment:
no need for this, you can change the volume with a params field in the sound, i.e:
```
        insertSound:
          path: /Audio/Effects/Vehicle/vehiclestartup.ogg
          params:
            volume: -3
```

Reply:
I tried to use this, but I noticed that it does not have much effect. So I had to make it quieter in the file itself.

Reply:
Try making it even quieter?